setup.py

```diff
@@ -55,6 +55,30 @@
 # Add a group made up of all optional dependencies
 extras_require['all'] = set(package for group in extras_require.values() for package in group)
 
+
+MODULES_TO_COMPILE = [
+    'nltk.grammar',
+    'nltk.parse.chart',
+]
```

Comment:
The CI passed! Lets see how far we can push this. Try adding these first: `nltk.classify`, `nltk.tag`,  `nltk.tokenize` and `nltk.translate`.